shared/chat/conversation/messages/wrapper/wrapper-author/container.js

```diff
@@ -12,8 +12,8 @@ const mapStateToProps = (state: TypedState, {message, previous, innerClass, isEd
   const isYou = state.config.username === message.author
   const isFollowing = state.config.following.has(message.author)
   const isBroken = state.users.infoMap.getIn([message.author, 'broken'], false)
-  const lastReadMessageID = state.chat2.lastReadMessageMap.get(message.conversationIDKey)
-  const lastPositionExists = !!previous && lastReadMessageID === previous.id
+  const {readMsgID, maxMsgID} = Constants.getMeta(state, message.conversationIDKey)
```

Comment:
Some please correct me if I'm wrong, but it seems like `Constants.getMeta` is the preferred way to get values from the chat2 metaMap?